3 - PB/Documentazione esterna/Manuale Utente/Manuale Utente.typ

```diff
@@ -58,14 +58,58 @@ Il prodotto offre le seguenti funzionalità principali:
 
 = Requisiti
 
+Di seguito sono elencati i requisiti minimi necessari per l'esecuzione dell'applicazione, comprese le caratteristiche necessarie per configurare l'ambiente di sviluppo del progetto.
+
 == Requisiti di sistema minimi
 
-== Requisiti di sistema consigliati
+#figure(
+  table(
+    columns: 3,
+    [*Componente*], [*Versione*], [*Riferimenti*],
+
+    [Docker],[>= 24.0.7],[https://docs.docker.com/],
```

Comment:
In tutta la tabella mettere i ">=" tra due $, in modo che venga visualizzato con il simbolo corretto